lib/matplotlib/legend.py

```diff
@@ -1108,6 +1115,12 @@ def _find_best_position(self, width, height, renderer, consider=None):
         assert self.isaxes
 
         verts, bboxes, lines, offsets = self._auto_legend_data()
+        if self._loc_used_default and verts.shape[0] > 200000:
+            # this size results in a 3+ second render time on a good machine
+            cbook._warn_external(
+                'Creating legend with loc="best" can be slow with large'
+                ' amounts of data.'
```

Comment:
```suggestion
                'amounts of data.'
```
We usually use trailing spaces with implicit string concatenation.